assets/mods/simplify/mod.js

```diff
@@ -434,20 +434,21 @@
 				}
 
 				const optionName = 'modEnabled-' + mod.name.toLowerCase();
-				const modOption = this.options.addEntry(optionName, 'CHECKBOX', true, tab, undefined, true);
+				const modOption = this.options.addEntry(optionName, 'MOD', true, tab, undefined, true);
 				// checkboxRightAlign is a custom field, see _hookRow
 				modOption.checkboxRightAlign = true;
 
 				const name = mod.displayName || mod.name;
 				const description = infoBoxSupported
 					? mod.description || ' '
 					: (mod.description || 'If checked this mod is enabled.') + ' \\c[1]Needs a restart!';
+				let icon = "";
+				if (mod.icon && mod.icon["24x24"]) {
```

Comment:
Allowing only a specific icon size to be used in the mod list is kind of strange. I'd use icon designations (such as `options-menu`) instead of their sizes for differentiating them in the `icon` object.

Reply:
I wanted to follow something similar to [https://developer.chrome.com/docs/extensions/mv2/manifest/icons/](this).